src/_utils/branding.ts

```diff
@@ -167,6 +168,22 @@ export const color: Color = {
   tapHighlight: 'rgba(221, 221, 221, .4)', // gray, 40%
 }
 
+
+export enum ResponsiveMode {
+  SMALL = 'small',
+  LARGE = 'large',
+}
+
+export const computeResponsiveModeClass = (mode:ResponsiveMode) => {
+  switch(mode) {
+    case ResponsiveMode.LARGE:
+      return 'rm-large'
```

Comment:
nit: usually I map `rm` to `remove` so it took me few seconds to understand ;)

Reply:
should I use full responsive-mode-large/small ?

Reply:
or `responsive-large` as you prefer

Reply:
+1, will do (If we keep these modes - cf the other discussion)